scripts/jenkins-test

```diff
@@ -214,9 +214,8 @@ then
 	    
             # we can run the python build, now that we have a spark executable
 	    mvn -U \
-		-P python,r \
-		package \
-		-DskipTests \
+		-P python \
+		test \
```

Comment:
Don't we need `package` for the line `./bin/pyadam < scripts/jenkins-test-pyadam.py` below (`package` includes `test`)?

Reply:
Doesn't seem like it, the tests are passing currently.

Reply:
Yeah, we shouldn't need it.